pages/getting-access.md

```diff
@@ -1,23 +1,71 @@
+communities. You must create an individual CCR account and enable two factor
+authentication if you wish to access any CCR resources. Please note that CCR
+user accounts are separate from your UBIT or Roswell Park accounts. 
+
+A CCR user account can be created quickly and easily by [by filling out the
+form here](https://idm.ccr.buffalo.edu/signup) using your UB or Roswell Park
+email address. Once your account is created you will need to verify your email
+address and enable two factor authentication. 
+
+## Enable Two Factor Authentication
+
+CCR requires all user accounts to have two factor authentication enabled. We
+support OTP, which stands for One-Time Passcodes and is a common form of two
+factor authentication (2FA). OTP is also known as app based authentication,
+software tokens, or soft tokens where an app on your phone will generate unique
+numeric passcodes as a second factor to provide increased account security.
+
+To enable two factor authentication on your account, follow these easy steps:
+
+1. Install an authenticator app on your phone. Authentication apps like Authy,
+   Google Authenticator, and Duo are supported. We recommend Duo as it's also used by UBIT.
+
+2. Login to CCR IDM portal [here](https://idm.ccr.buffalo.edu/) and click on
+   [OTP Tokens](https://idm.ccr.buffalo.edu/otp) in the side menu.
+
+3. Click on the "New Token" button and follow the directions to scan the QR
+   code using your authenticator app. Enter the six digit passcode generated by the app.
+
+You can also watch the video below which details the entire process:
+
+![type:video](https://youtube.com/embed/BDd-J1DAwsw)  
+
+## Allocation Requests
+
+Access to CCR resources is managed using allocations. ColdFront is the
+allocations management system CCR uses to manage resource allocations for the
+entire center. In order to access CCR resources you must be on an active
+allocation. 
 
-## Requesting an Allocation in ColdFront
+- __Faculty members__ (Principal investigators) can create projects in ColdFront,
+  add group members, and request allocations to resources.  Once an allocation
+  is activated, users on that allocation will be able to access the resource.
+  More details about ColdFront can be found [here](portals/coldfront.md).
 
-ColdFront is CCR's resource and allocation management system.  ColdFront manages all of CCR's resources and the access all users have to those resources.  Faculty members (Principal investigators) must create themselves a project in ColdFront, request allocations for the resources their group wants to use, and then add their group members to their project.  Once an allocation is activated, users on that allocation will be able to access the resource.  [More details about ColdFront can be found here](portals/coldfront.md).
+- __Students, postdocs, and external collaborators__ can contact the faculty member
+  (PI) who is sponsoring your account to request that they add you to their
+  project and allocations.
```

Comment:
I would prefer to leave the external collaborator information as a separate bullet.  Telling them to contact their PI sponsor who may not know what to do isn't as helpful, IMO

Reply:
Good point.. all fixed.